From: Soeren Moeller <soerenmoeller2001@gmail.com>
Date: Fri, 14 May 2010 19:03:00 +0000 (+0000)
Subject: Staging: udlfb: fix coding style issues

commit b5a2104c98cb603f7053e4b0309fb88f15d6be86 upstream.

This is a patch to the file udlfb.c that fixes a missing KERN_INFO and
removes one whitespace before a newline.

Signed-off-by: Soeren Moeller <soerenmoeller2001@gmail.com>
Cc: Bernie Thompson <bernie@plugable.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

diff --git a/drivers/staging/udlfb/udlfb.c b/drivers/staging/udlfb/udlfb.c
index 12444f2..7fc1d61 100644
--- a/drivers/staging/udlfb/udlfb.c
+++ b/drivers/staging/udlfb/udlfb.c
@@ -1438,7 +1438,7 @@ static int __init dlfb_module_init(void)
 	if (res)
 		err("usb_register failed. Error number %d", res);
 
-	printk("VMODES initialized\n");
+	printk(KERN_INFO "VMODES initialized\n");
 
 	return res;
 }
@@ -1567,7 +1567,7 @@ static int dlfb_alloc_urb_list(struct dlfb_data *dev, int count, size_t size)
 
 	kref_get(&dev->kref); /* released in free_render_urbs() */
 
-	dl_notice("allocated %d %d byte urbs \n", i, (int) size);
+	dl_notice("allocated %d %d byte urbs\n", i, (int) size);
 
 	return i;
 }
